meta/entity/draft-1.json

```diff
@@ -8,7 +8,19 @@
     {
       "type": "object",
       "properties": {
-        "aliases": {
+        "Name": {
+          "type": "string",
+          "description": "Preferred gene name"
+        },
+        "Taxon id": {
```

Comment:
`Gene ID`
`Taxon id`

Let's use either `ID` or `id` consistently; perhaps normalize to `ID`